cloudformation-prod.json

```diff
@@ -96,7 +94,7 @@
 		"Ec2Instance" : {
       "Type" : "AWS::EC2::Instance",
       "Properties" : {
-        "ImageId" : "ami-0d77397e",
+        "ImageId" : { "Fn::FindInMap" : [ "RegionMap", { "Ref" : "AWS::Region" }]},
         "KeyName" : { "Ref" : "KeyName" },
 				"InstanceType" : "t2.micro",
```

Comment:
If/when we need to beef up the EC2 instance, am I correct to assume this is where that change would occur?